rosetta-source/src/main/resources/cdm-sample-files/functions/repo-and-bond/repo-adjustment.md

```diff
@@ -0,0 +1,12 @@
+### Create Repo Reprice Business Event
```

Comment:
Is the detail in this .md correct? Are we using the reprice as an example of an adjustment?

Reply:
fixed